keyboards/pimoroni/keybow2040/info.json

```diff
@@ -0,0 +1,59 @@
+{
+    "manufacturer": "Pimoroni",
+    "keyboard_name": "Keybow 2040",
+    "url": "https://shop.pimoroni.com/products/keybow-2040",
+    "maintainer": "fanf2",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "features": {
+	"console": true
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x16D0",
+        "pid": "0x08C6",
+    },
+    "eeprom": {
+	"driver": "transient",
+    },
```

Comment:
The default EEPROM driver (which for RP2040 emulates EEPROM using a part of the same flash chip that contains the firmware) should work (at least the official schematic shows the W25Q16JVUXIQ chip, which should be supported).  Enabling the VIA support with the `transient` driver is mostly useless.
```suggestion
```

Reply:
Yes indeed :-) I left that in there as a debugging aid because I have been *very* confused by unexpected statefulness in the past! Sadly json does not let me explain things like this in the best place :-/

Reply:
Technically `info.json` files are parsed as Hjson by the Python code and therefore allow comments and some other syntax extensions, but there is some disagreement about whether the usage of such things should actually be encouraged.

However, `keymap.json` files must be pure JSON (one reason is that QMK Configurator can fetch those files directly from the `qmk_firmware` repo in some cases).

Reply:
I've removed the debugging remnants from round two